experiments/training.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from pathlib import Path
+from transfer_nlp.plugins.config import ExperimentConfig
+from experiments.surnames import *
+from experiments.news import *
+from experiments.cbow import *
+
+name = 'transfer_nlp.experiments.training'
+logging.getLogger(name).setLevel(level=logging.INFO)
+logger = logging.getLogger(name)
+
+if __name__ == "__main__":
+    logging.basicConfig(level=logging.INFO)
+
+    surname_paths = ['/Users/petermartigny/Documents/PycharmProjects/transfer-nlp/experiments/mlp.json',
```

Comment:
shouldn't check in user directories into git. probably not pycharm either. 
maybe use some very generic convention like `str(Path.home() / 'work/transfer_nlp'`?

Reply:
This file should be run from the `experiments/` folder so I changed them to `"./experiment_name"`